examples/fast_neural_style/neural_style.py

```diff
@@ -0,0 +1,291 @@
+# coding: utf-8
+from __future__ import print_function, division
+
+import argparse
+import os
+import sys
+import time
```

Comment:
Same here, it seems that time, re are not used, according to PyCharm

Reply:
Incorporated